### PR TITLE
Fix "source" branch `polkadot-v.9.27`

### PR DESCRIPTION
**Pull request by chevdor**
- fast-forward to current master HEAD
- switch to release branches
- bump versions
- cargo update
- fix tx_version due to https://github.com/paritytech/cumulus/pull/1494 not being merged yet

### CODE REVIEW

**Review by alvicsam — APPROVED** [review]

**Review by shawntabrizi — APPROVED** [review]
did not really review, but assume this is sane.

**Comment by chevdor** [comment]
Thanks for your support @shawntabrizi :)
The build fails so there is still something to be fixed before this is really mergeable.

**Review by sergejparity — APPROVED** [review]

File: parachains/runtimes/assets/westmint/src/lib.rs [hunk 1]
```diff
@@ -83,7 +83,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("westmint"),
 	impl_name: create_runtime_str!("westmint"),
 	authoring_version: 1,
-	spec_version: 9220,
+	spec_version: 9270,
 	impl_version: 0,
 	apis: RUNTIME_API_VERSIONS,
 	transaction_version: 6,
```
Comment (NachoPal) [inline parachains/runtimes/assets/westmint/src/lib.rs:89]:
```suggestion
	transaction_version: 7,
```

**Review by NachoPal — COMMENTED** [review]
Besides some CI stuff I can not tell is correct or not, it LGTM.

**Pull request merged**